mabwiser/simulator.py

```diff
@@ -242,15 +247,25 @@ def _get_nhood_predictions(self, lp, row_2d, indices, is_predict):
 
             return prediction, {}, arm_to_stat
 
+    def _get_no_nhood_predictions(self, lp, is_predict):
```

Comment:
could this be imported instead of duplicate?

Reply:
Good catch!